OpenDreamRuntime/Objects/MetaObjects/DreamMetaObjectRoot.cs

```diff
@@ -1,10 +1,17 @@
-﻿using OpenDreamRuntime.Procs;
+using OpenDreamRuntime.Procs;
+using OpenDreamShared.Dream;
 
 namespace OpenDreamRuntime.Objects.MetaObjects {
     [Virtual]
     class DreamMetaObjectRoot : IDreamMetaObject {
```

Comment:
I don't see a point in there being a `DreamMetaObjectRoot` anymore. You could have every meta object implement `IDreamMetaObject` directly instead.

Reply:
Because it reduces code duplication